Assets/_Application/Samplers/PointerToWorldPosition.cs

```diff
@@ -14,7 +14,7 @@ public class PointerToWorldPosition : MonoBehaviour
         private OpticalRaycaster opticalRaycaster;
         private Action<Vector3, bool> worldPointCallback;
         private Coordinate worldPoint;
-        private float maxDistance = 10000;
+        private float maxDistance = 1000;
```

Comment:
Waarom is dit nu 10x minder? dat is volgens mij niet wenselijk, want dan kan je niet heel ver klikken